allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureLabelsTest.java

```diff
@@ -0,0 +1,58 @@
+package io.qameta.allure.examples.junit5;
+
+import io.qameta.allure.Allure;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import io.qameta.allure.LabelAnnotation;
+import io.qameta.allure.Owner;
+import io.qameta.allure.Story;
+import org.junit.jupiter.api.Test;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+public class AllureLabelsTest {
+
+    @Test
+    @Epic("static epic")
+    @Feature("static feature")
+    @Story("static story")
+    @Owner("static owner")
+    public void commonStaticLabelTest() {
+    }
+
+    @Test
+    public void commonDynamicLabelTest() {
+        Allure.epic("dynamic epic");
+        Allure.epic("dynamic feature");
+        Allure.epic("dynamic story");
+        Allure.epic("dynamic owner");
+    }
+
+    @Test
+    @CustomLabel("static value")
```

Comment:
I would say (though this might be subjective) that the more useful example would be to create @CustomLabel without params. 

In real life supporting code like: 

```
@MySpecial
@Test
void somethingVeryImportant() {
    assert 2 + 2 == 4;
}
```

... would be much easier than


```
@CustomLabel("my special")
@Test
void somethingVeryImportant() {
    assert 2 + 2 == 4;
}
```

For sure there are usecases for custom labels with params, but probably it would be more natural to start an easier example of probably more handy usage...